src/main/core/support/configuration.rs

```diff
@@ -1045,6 +1045,13 @@ impl From<nix::sys::signal::Signal> for Signal {
     }
 }
 
+impl From<linux_api::signal::Signal> for Signal {
+    fn from(value: linux_api::signal::Signal) -> Self {
+        let signal = nix::sys::signal::Signal::try_from(value.as_i32()).unwrap();
+        Self(signal)
+    }
+}
```

Comment:
Done, though it just moves the unwrap up to the one point where it's used :). Probably makes more sense there, though.

We currently only use this to convert from a signal that a process actually died of, into the config format for display. The conversion currently can't fail since we don't support sending realtime signals in the first place.

It's probably be better to change the config format to use linux_api's Signal, but then we'd need to implement conversions to/from string while being careful to preserve compatibility. Added a comment.